apple/Sources/Sargon/Extensions/Methods/Profile/ProfileFileContents+Wrap+Functions.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  File.swift
+//  
+//
+//  Created by Alexander Cyon on 2024-05-01.
+//
+
+import Foundation
+import SargonUniFFI
+
+extension ProfileFileContents {
+	public func hash(into hasher: inout Hasher) {
+		hasher.combine(profileFileContentsHashValue(contents: self))
```

Comment:
`ProfileFileContents ` is an enum with the associated value of the `PlaintextProfile` being `RefProfile` - a reference type. Thus UniFFI cannot make `ProfileFileContents` conform to `Hashable` / `Equatable`. We have to build it ourselves. I've implemented in `Eq` and `Hash` in Rust and UniFFI exported it...